common/src/main/java/com/microsoft/identity/common/internal/cache/ActiveBrokerCacheUpdater.kt

```diff
@@ -76,20 +85,42 @@ class ActiveBrokerCacheUpdater(
             bundle.putString(ACTIVE_BROKER_SIGNING_CERTIFICATE_THUMBPRINT_KEY,
                 activeBroker.signingCertificateThumbprint)
         }
+
+        /**
+         * Adds the "broker discovery is now disabled" to the result bundle.
+         *
+         * @param bundle       The result bundle.
+         */
+        @JvmStatic
+        fun appendBrokerDiscoveryDisabledToResultBundle(bundle: Bundle) {
```

Comment:
Currently we seem to only use this in testing, do we need to also call before returning a result somewhere in our broker operations?

Reply:
yes. in this PR https://github.com/AzureAD/ad-accounts-for-android/pull/2485